pydocx/export/__init__.py

```diff
@@ -4,10 +4,11 @@
     unicode_literals,
 )
 
-from pydocx.export.html import PyDocXHTMLExporter
+from pydocx.export.html import PyDocXHTMLExporter, PyDocXHTMLExporterWithImageResize
```

Comment:
I like that idea. What do you think about making that a separate pull request? I think it would be good to offer that suggestion to the maintainers of PyDocX, but I wouldn't want them to reject the core mixin pull-request because of it.

It may turn out that they think it's a good idea to just resize by default. I don't really see why you wouldn't want to do it every time.